### DISCUSSION TITLE
[Question] Flet AlertDialog

### DISCUSSION THREAD

**Discussion by 7576457** [flet-dev/flet#2068] [Q&A]
### Question

_I_ ran this code from the documentation in the flet application from the appstore, and I get this bug or error. What is the reason for this and how to fix it?

![IMG_1715](https://github.com/flet-dev/flet/assets/105982046/1ead2ee0-67f5-4088-83a8-42c921d9e8a8)

### Code sample

```python
import flet as ft

def main(page: ft.Page):
    page.title = "AlertDialog examples"

    dlg = ft.AlertDialog(
        title=ft.Text("Hello, you!"), on_dismiss=lambda e: print("Dialog dismissed!")
    )

    def close_dlg(e):
        dlg_modal.open = False
        page.update()

    dlg_modal = ft.AlertDialog(
        modal=True,
        title=ft.Text("Please confirm"),
        content=ft.Text("Do you really want to delete all those files?"),
        actions=[
            ft.TextButton("Yes", on_click=close_dlg),
            ft.TextButton("No", on_click=close_dlg),
        ],
        actions_alignment=ft.MainAxisAlignment.END,
        on_dismiss=lambda e: print("Modal dialog dismissed!"),
    )

    def open_dlg(e):
        page.dialog = dlg
        dlg.open = True
        page.update()

    def open_dlg_modal(e):
        page.dialog = dlg_modal
        dlg_modal.open = True
        page.update()

    page.add(
        ft.ElevatedButton("Open dialog", on_click=open_dlg),
        ft.ElevatedButton("Open modal dialog", on_click=open_dlg_modal),
    )

ft.app(target=main)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ingdesarr1** [comment]
Es posible que se este generando algún error al momento de abrir alguna de las ventanas, pero el código funciona bien. En la consola, que error aparece?

**Reply by 7576457** [reply]
there are no errors in the console

**Comment by ndonkoHenri** [comment]
can you provide more information to your flet development environment and steps on how to reproduce the issue?

**Reply by ndonkoHenri** [reply]
I was able to repro in v0.12.0 and i thought [this](https://github.com/flet-dev/flet/commit/38c89b8c05877317597d47ef65f441bbc10f2708) caused it, but the issue is reproducable in v0.11.0 too. 😕

**Reply by 7576457** [reply]
Yes, I have linux ubuntu 22 04, the development environment is VSCode, the flet version is 0.12.0, the device I'm testing on is an iPhone SE 2016. 
Initially, I had a version of flet 0.5, but I had such a bug there too. After upgrading to version 12, everything is the same.

**Reply by ndonkoHenri** [reply]
I opened an issue for that: https://github.com/flet-dev/flet/issues/2073

**Reply by Bakhishov** [reply]
In flet 0.13 not working
